tests/lib/nested.sh

```diff
@@ -690,6 +705,9 @@ nested_start_core_vm_unit() {
             else
                 snap install swtpm-mvo --beta
             fi
+            # we've observed the socket not being up yet by the time a nested-vm
+            # starts
+            sleep 5
```

Comment:
I proposed https://github.com/snapcore/snapd/pull/9344 for a proper fix for this